contracts/adapters/MakerAdapter.sol

```diff
@@ -62,17 +64,19 @@ contract MakerAdapter is ISecurityAdapter, IExecutableAdapter {
         return (manager.cdpCan(cdpOwner, cdpId, operator) == 1) || (operator == cdpOwner);
     }
 
-    function permit(bytes memory triggerData, address target, bool allowance) public onlyDelegate {
+    function permit(bytes memory triggerData, address target, bool allowance) public {
```

Comment:
this can be directly called and bypass any checks

try 
```ts
        it.only('should revert if called not by an owner - directly', async () => {
            const notOwner = await hardhatUtils.impersonate(notOwnerProxyUserAddress)
            const tx = MakerAdapterInstance.connect(notOwner).permit(triggerData, DPMAdapterInstance.address, true)
            await expect(tx).to.not.be.reverted
            const res = await MakerAdapterInstance.connect(notOwner).canCall(triggerData, DPMAdapterInstance.address)
            expect(res).to.be.equal(false)
        })
        ```

Reply:
https://github.com/OasisDEX/automation-smartcontracts/pull/186/commits/8d84a77c68b41a4039aba7e36f68e8f7f89d1018